src/stories/HeatmapVis.stories.tsx

```diff
@@ -71,6 +89,7 @@ Alpha.args = {
   alphaDomain,
 };
 
+export { Template as HeatmapVisTemplate };
```

Comment:
Exporting the `Template` to reuse the specific gamma controls in other Heatmap stories files.

Note that I use `excludeStories` below to remove it from the list of stories of this file.